jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -730,9 +731,21 @@ static Instant getCreationTime(String configuredCreationTime, ProjectProperties
           return Instant.now();
 
         default:
-          return DateTimeFormatter.ISO_DATE_TIME.parse(configuredCreationTime, Instant::from);
+          DateTimeFormatter formatter =
+              new DateTimeFormatterBuilder()
+                  .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+                  .optionalStart()
+                  .appendOffset("+HH:MM", "+00:00")
+                  .optionalEnd()
+                  .optionalStart()
+                  .appendOffset("+HHMM", "+0000")
+                  .optionalEnd()
+                  .optionalStart()
+                  .appendOffset("+HH", "Z")
+                  .optionalEnd()
+                  .toFormatter();
+          return formatter.parse(configuredCreationTime, Instant::from);
```

Comment:
Should we support `-HHMM` and `-HH:MM` as well? And I am not sure if `+` in `+HH` is needed. Or perhaps I misunderstood the meaning of `+`.

And should we support [+hh:mm:ss and -hh:mm:ss](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneOffset.html#getId--) as well? I don't expect anyone to set a second for the offset, but a machine may generate it for completeness.

It's also worth making a CHANGELOG entry that Jib no longer accepts the [non-ISO8601 zone format](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME) (e.g., `[Europe/Paris]`)? Or, should we just use this new formatter only when the built-in formatter fails?

Reply:
That said, it's worth adding tests using `-`.

Reply:
Yeah I tested them to see it work, but I'll add them to the tests.

Reply:
hrmm... maybe I can accept the zone format too. I'll try not to break anything.

Reply:
So it seems like the formatter will accept seconds, but just ignore them?

Reply:
It's not completely ignored. The following code prints `1322842530` and `1322842529` respecively.

```java
    System.out.println(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2011-12-03T01:15:30+09:00:00")).getEpochSecond());
    System.out.println(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2011-12-03T01:15:30+09:00:01")).getEpochSecond());
```

However, I don't really think anyone will really use seconds (other than `00`) as part of timezone.

Reply:
right because of the ISO_DATE_TIME formatter using `HH:MM:ss`, but for `HHmm` seconds is not allow (I thought it was ignored, but that is untrue). In any case, I can't think of a timezone that is offset by seconds.